variables.tf

```
@@ -68,6 +68,18 @@ variable "redis_timeout" {
   default     = "10m"
 }
 
+variable "read_replicas_enabled" {
+  description = "Whether to enable redis replicas or not. Read replica mode. Can only be specified when trying to create the instance. If set to true, memory_size_gb variable value should be 5 or above"
+  type        = bool
+  default     = false
+}
+
+variable "replica_count" {
```

It's helpful to use common prefix for similarly grouped items. Helps to search words more effectively later.
```suggestion
variable "read_replicas_count" {
```